api/pkg/api/main.go

```diff
@@ -145,7 +145,8 @@ func saveHarToDb(entry *har.Entry, connectionInfo *tap.ConnectionInfo) {
 	}
 	database.GetEntriesTable().Create(&mizuEntry)
 
-	baseEntry := utils.GetResolvedBaseEntry(mizuEntry)
+	baseEntry := models.BaseEntryDetails{}
+	_ = models.GetEntry(&mizuEntry, &baseEntry)
```

Comment:
Do not ignore errors.
Don't try to use `baseEntry` if `GetEntry` returned an error.